bluemira/equilibria/find_legs.py

```diff
@@ -66,7 +77,7 @@ def __init__(
         eq: Equilibrium,
         psi_n_tol: float = 1e-6,
         delta_start: float = 0.01,
-        rtol: float = 1e-1,
+        rtol: float = 1e-3,
```

Comment:
Can you explain this change?

Reply:
Setting it to a lower tolerance was part of some testing that was left in, I am returning it to a more strict default.